src/main.js

```diff
@@ -1,11 +1,14 @@
 import ElementUI from 'element-ui';
+import authStorage from '@/utils/auth';
 import Vue from 'vue';
 import i18n from '@/lang';
 import App from './App';
 import router from './router';
 import store from './store';
 import './config';
 
+// 初始化使用普通登录
+authStorage.setIsSsoLoginInfo(false);

```

Comment:
the auth.js is already configured in vue.config.js plugins.

please to use $auth.setIsSsoLoginInfo(false) to instead

tks